### ISSUE TITLE
In situations where the missing rate is less than 0.1

### ISSUE THREAD

**Issue by ZLCL** [FMLYD/psw-i#3]
First of all, thank you for your work.
I utilized data containing 60,000 timestamps and 21 feature columns. For comparative experiments, I accidentally adjusted the missing rate to below 0.1, such as 0.05, 0.01,1e-07, etc. The experimental results under these conditions are as follows:
Round0 - sinkhorn on Arou_raw: MAE=0.6025, MSE=0.8806, MRE=7.3755
Done! Final results:
MAE=0.6025 , MSE=0.8806 , MRE=7.3755 , average inference train_model.py=0.00
and
Round0 - sinkhorn on Arou_raw: MAE=0.4909, MSE=0.6707, MRE=1.7393
Done! Final results:
MAE=0.4909 , MSE=0.6707 , MRE=1.7393 , average inference train_model.py=0.00
and
Round0 - sinkhorn on Arou_raw: MAE=0.6487, MSE=0.9687, MRE=59300456310099496.0000
Done! Final results:
MAE=0.6487 , MSE=0.9687 , MRE=59300456310099496.0000 , average inference train_model.py=0.00

It can be observed that the effect is poor and the experiment stops after only one round. What is the reason for this.

**Comment by FMLYD** [comment]
Hi, thanks for your interest in our work.

Due to limited time and the lack of detailed information about your dataset, we’re currently unable to support experiments on unknown or private datasets. Our method has only been validated on publicly available benchmarks.

From the results you've provided, it seems that the variance across runs is relatively high. We recommend adjusting some key hyperparameters — such as the patch length, learning rate, etc. — to better fit your specific dataset.

Thanks for your understanding!

**Comment by ZLCL** [comment]
Thank you for your response. I will continue to try. Thank you again for your work.
